crates/flycheck/src/lib.rs

```diff
@@ -108,7 +108,7 @@ struct FlycheckActor {
 
 enum Event {
     Restart(Restart),
-    CheckEvent(Option<cargo_metadata::Message>),
+    CheckEvent(Option<Box<cargo_metadata::Message>>),
```

Comment:
I think boxing will incur performance degradation instead of increase since 99% of the events are `CheckEvent`. Heap allocating them all will hurt, let's leave it as is even though the size difference between `Restart` and `Option<cargo_metadata::Message>` is big,

Reply:
Sure!